CONTRIBUTING.md

```diff
@@ -136,10 +138,11 @@ npm run cypress:run
 - Reference [Issues](https://github.com/EddieHubCommunity/LinkFree/issues) and [Pull Requests](https://github.com/EddieHubCommunity/LinkFree/pulls) liberally after the first line.
 
 ---
+
 ## 💥 Issues
 
 In order to discuss changes, you are welcome to [open an issue](https://github.com/EddieHubCommunity/LinkFree/issues/new/choose) about what you would like to contribute. Enhancements are always encouraged and appreciated.
 
 ## All the best! 🥇
 
-[![built with love](https://forthebadge.com/images/badges/built-with-love.svg)](eddiehub.org)
+[![built with love](https://forthebadge.com/images/badges/built-with-love.svg)](https://www.eddiehub.org/?r_done=1)
```

Comment:
I'd lead to the EddieHub page without the `r_done` part; it makes it look better, IMO.
```suggestion
[![built with love](https://forthebadge.com/images/badges/built-with-love.svg)](https://www.eddiehub.org/)
```

Reply:
Thanks for the review. 
Open source is fun